src/app/entities/courses/components/course-item/course-item.component.html

```diff
@@ -24,7 +24,7 @@ <h2 class="card__title">{{ course?.title | uppercase }}</h2>
     <div class="course-item__body">
       <div class="course-item__description">{{ course?.description }}</div>
       <div class="course-item__actions">
-        <button class="button button--small button--blue">
+        <button class="button button--small button--blue" (click)="onCourseEditClick(course.id, course.title)">
```

Comment:
just pass a `course` object and get all necessary props...

Reply:
Okay